cmd/list.go

```diff
@@ -23,35 +22,27 @@ type role struct {
 type roles map[role]struct{}
 
 type whoCan struct {
-	verb         string
-	resource     string
-	resourceName string
-	client       kubernetes.Interface
-	r            roles
-}
+	verbArg         string
+	resourceArg     string
+	resourceNameArg string
```

Comment:
I’m not sure I love the name changes on these fields, I think it makes it slightly harder to read - what’s your reasoning here?

Reply:
I tend to agree. It came from the fact that I wanted to better distinguish the raw `resource` arg from the actually resolved apiResource of type `APIResource`. Anyhow `resource` and `apiResource` should be fine. I will revert it while rebasing/resolving conflicts.